src/pages/AuthPages/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import "./Auth.css"
+import Navbar from "../../components/Navbar/Navbar"
+import { Link } from 'react-router-dom';
+
+function Login() {
+  return (
+    <>
+      <Navbar />
+
+       <section className="auth-section">
+          <div className="auth-container">
+            <h3>Login</h3>
+            <br />
+            <form>
+              <div className="input-container">
+                <label className="input-label">Email Address*</label>
+                <input className="input-field" type="text" required />
+              </div>
+              <br />
+              <div className="input-container">
+                <label className="input-label">Password*</label>
+                <input className="input-field" type="password" required />
+              </div>
+              <br />
```

Comment:
Avoid using `br` tags as it is a bad practise. Use margin and padding instead to give spacing